qutebrowser/completion/models/miscmodels.py

```diff
@@ -48,8 +51,14 @@ def __init__(self, parent=None):
         for name, cmd in config.section('aliases').items():
             cmdlist.append((name, "Alias for '{}'".format(cmd)))
         cat = self.new_category("Commands")
+
+        # map each command to its bound keys and show these in the misc column
+        keyconf = objreg.get('key-config')
+        cmd_to_keys = defaultdict(lambda: [])
+        for key, cmd in keyconf.get_bindings_for('normal').items():
+            cmd_to_keys[cmd].append(key)
         for (name, desc) in sorted(cmdlist):
-            self.new_item(cat, name, desc)
+            self.new_item(cat, name, desc, str.join(',', cmd_to_keys[name]))
```

Comment:
I think adding a space after the comma would be nicer, and there should be enough space for it?